packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openstack/openstackSecretFieldValidator.ts

```diff
@@ -269,20 +349,10 @@ const validateApplicationCredentialName = (value: string): ValidationMsg => {
   return { type: 'error', msg: 'Invalid application name, spaces are not allowed' };
 };
 
-const validateInsecureSkipVerify = (value: string): ValidationMsg => {
```

Comment:
why don't we need this anymore ?

Reply:
It was implemented only for openStack and seems redundant since the user can't really set it to an invalid field via the ui (not an input text field). All other providers included a successful validation result only. 

This is not part of this PR fix, but I anyway fixed that by adding that validation for all providers now.